### PR TITLE
fix: accessibility of forms and save buttons

### PR DESCRIPTION
**Pull request by Nikomus**
https://trello.com/c/qSA68552/117-quiz-question-accessibility

- Checkboxes should now announce what their questions are
- Slightly better names for repeated button controls

### CODE REVIEW

File: pages/quiz/whats-on-your-mind.tsx [hunk 1]
```diff
@@ -87,39 +86,37 @@ export const WhatsOnYourMindPage = (): JSX.Element => {
...
-        <Navigation>
-          <LinkButton
-            textContent="skip"
-            arrow="forward"
-            onClick={() => router.push('how-are-you-feeling')}
-          />
-        </Navigation>
-        <QuestionSection>
-          <h2>Choose as many as you like</h2>
+      <HeaderComponent title="Support in Lambeth" />
```
Comment (edpe) [inline pages/quiz/whats-on-your-mind.tsx:89]: The header component for the quiz welcome page displays 'Quiz' and then switches to 'Support in Lambeth' for the actual questions. 

It also seems confusing for questions which are about how the user feels, what's on their mind etc. to have a heading that isn't related to the questions they are being asked. The app name would be ok, but 'Support in Lambeth' seems inconsistent with the rest of the page content.

File: pages/quiz/about-you.tsx [hunk 2]
```diff
@@ -99,17 +121,17 @@ export const AboutYouPage = (): JSX.Element => {
...
-          />
-        </Navigation>
-        <QuestionSection>
-          <h2>How old are you?</h2>
+      <HeaderComponent title="Support in Lambeth" />
+      <Navigation>
+        <LinkButton
+          textContent="back"
+          arrow="back"
+          onClick={() => router.back()}
```
Comment (edpe) [inline pages/quiz/about-you.tsx:129]: This can be a link to the 'what-are-your-interests' page, I think I missed it in my last PR.

**Review by edpe — APPROVED** [review]
Looks better and generally more accessible :eagle:

**Review by jthistle — APPROVED** [review]
Looks good to me

File: pages/quiz/whats-on-your-mind.tsx (hunk 1 above)
Comment (Nikomus) [reply pages/quiz/whats-on-your-mind.tsx:89]: I think "Support in Lambeth" is the de facto title of the site at the moment - I'll make that the title throughout the quiz for now.

**Comment by marxian** [comment]
#### 📝Changed routes:


| [**/filter**](https://my-best-life-lm47o5tgp-neontribe.vercel.app/filter) | [**/quiz/about-you**](https://my-best-life-lm47o5tgp-neontribe.vercel.app/quiz/about-you) |
|:-:|:-:|
|<a href="https://my-best-life-lm47o5tgp-neontribe.vercel.app/filter">Screenshot of /filter</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/JmnVBgQXx7jszPAcld7dDeDcmAk_.png">(view full size)</a> |<a href="https://my-best-life-lm47o5tgp-neontribe.vercel.app/quiz/about-you">Screenshot of /quiz/about-you</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/ofzr_fTHXUfsWz99Ew5JnZ66C0Y_.png">(view full size)</a> |



| [**/quiz/how-are-you-feeling**](https://my-best-life-lm47o5tgp-neontribe.vercel.app/quiz/how-are-you-feeling) | [**/quiz**](https://my-best-life-lm47o5tgp-neontribe.vercel.app/quiz) |
|:-:|:-:|
|<a href="https://my-best-life-lm47o5tgp-neontribe.vercel.app/quiz/how-are-you-feeling">Screenshot of /quiz/how-are-you-feeling</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/j_NC_M4xM1q1bbXloyCzTdlBGpk_.png">(view full size)</a> |<a href="https://my-best-life-lm47o5tgp-neontribe.vercel.app/quiz">Screenshot of /quiz</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/5TrFz_CQXZS3hx1UORtK2tWqtq8_.png">(view full size)</a> |



| [**…hat-are-your-interests**](https://my-best-life-lm47o5tgp-neontribe.vercel.app/quiz/what-are-your-interests) | [**/quiz/whats-on-your-mind**](https://my-best-life-lm47o5tgp-neontribe.vercel.app/quiz/whats-on-your-mind) |
|:-:|:-:|
|<a href="https://my-best-life-lm47o5tgp-neontribe.vercel.app/quiz/what-are-your-interests">Screenshot of /quiz/what-are-your-interests</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/xYqWfS0EVedil0F_2wmSKCmqYdE_.png">(view full size)</a> |<a href="https://my-best-life-lm47o5tgp-neontribe.vercel.app/quiz/whats-on-your-mind">Screenshot of /quiz/whats-on-your-mind</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/IYjdHRpRaTQOGJmrUiOKOetw5ak_.png">(view full size)</a> |



And 1 other route:
- [**/service/\[id\]**](https://my-best-life-lm47o5tgp-neontribe.vercel.app/service/[id])



Commit cc408f9d1c1d846272711b5987662b81fdfcd797 (https://my-best-life-lm47o5tgp-neontribe.vercel.app).

**Pull request merged**